src/lecturer/components/sendQuizView/SendQuizView.tsx

```diff
@@ -126,11 +124,11 @@ export function SendQuizView() {
                             {`Upewnij się, że wszystko jest prawidłowo ustawione i wyślij quiz:`}
                         </Typography>
                         <List>
-                        <ListItem>
+                            <ListItem>
                                 <ListItemText
                                     primary="Nazwa przydzielonego quizu:"
                                     secondary={
-                                        (store.quizes[store.selectedQuiz] != undefined) ? store.quizes[store.selectedQuiz].title : ""
+                                        store.quizes[store.selectedQuiz] ?? ""
```

Comment:
what does double question mark means? ("??")

Reply:
it is "shorter if" with fall back value if tested is "null" or "undefined"

https://developer.mozilla.org/pl/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator